feat(android): add serial number support for zebra devices on android… (#1)
MobisysGmbH/cordova-plugin-device: README.md

#         under the License.
-->

> # cordova-plugin-device (with zebra s/n support on Android 10+)

Reviewer: This plugin, the generated npm package and this repository should be renamed to `mobisys-cordova-plugin-device`. Also, there's no need to mention the feature in the title - the title should only contain the project name.

```suggestion
# mobisys-cordova-plugin-device
```

[![Android Testsuite](https://github.com/apache/cordova-plugin-device/actions/workflows/android.yml/badge.svg)](https://github.com/apache/cordova-plugin-device/actions/workflows/android.yml) [![Chrome Testsuite](https://github.com/apache/cordova-plugin-device/actions/workflows/chrome.yml/badge.svg)](https://github.com/apache/cordova-plugin-device/actions/workflows/chrome.yml) [![iOS Testsuite](https://github.com/apache/cordova-plugin-device/actions/workflows/ios.yml/badge.svg)](https://github.com/apache/cordova-plugin-device/actions/workflows/ios.yml) [![Lint Test](https://github.com/apache/cordova-plugin-device/actions/workflows/lint.yml/badge.svg)](https://github.com/apache/cordova-plugin-device/actions/workflows/lint.yml)

> ## Info

Reviewer: Should be reworded to "Notice", because "Info" is very generic.

```suggestion
## Notice
```

> This is a fork of the oficial [cordova-plugin-device](https://github.com/apache/cordova-plugin-device) adding the possibility to read out the serial number of a zebra device running on Android 10 or higher by using [DeviceIdentifiersWrapper](https://github.com/ZebraDevs/DeviceIdentifiersWrapper).

Reviewer: ```suggestion

This is a fork of the official [cordova-plugin-device](https://github.com/apache/cordova-plugin-device) with the goal to add the possibility to read the serial number of Zebra devices running on Android 10 or higher by using Zebra's [DeviceIdentifiersWrapper](https://github.com/ZebraDevs/DeviceIdentifiersWrapper).
```
